src/utils/managing.officer.individual.ts

```diff
@@ -11,7 +11,7 @@ import {
   mapFieldsToDataObject,
   prepareData,
   setApplicationData,
-  removeFromApplicationData
+  removeFromApplicationData,
```

Comment:
, not needed

Reply:
Admittedly I don't _always_ do it, but having trailing commas is better than not, mainly when it comes to merging, so going to leave this for now unless we take a hard yes/no approach to them.

Reply:
Resolving as all merge checks have passed and approval given.